Drop old Map Reduce code

milleruntime opened the pull request.

ctubbsii commented on hadoop-mapreduce/pom.xml:
```diff
@@ -94,7 +94,7 @@
             <configuration>
               <includes>
-                <include>org[.]apache[.]accumulo[.]core[.]client[.]mapred(?:uce)?[.].*</include>
+                <include>org[.]apache[.]accumulo[.]hadoop[.]mapred(?:uce)?[.].*</include>
```
Was this previously not analyzing classes in this module? That's weird.

ctubbsii reviewed (commented):
Looks good overall. I think you've already moved any old tests to the new M/R framework, but it's possible there could be some that were missed. This would be good to get into 3.0.

milleruntime commented:
> I think you've already moved any old tests to the new M/R framework, but it's possible there could be some that were missed.

I moved `MapReduceIT` in this PR. There are 2 other ITs that had tests that I dropped: `SslIT` and `SslWithClientAuthIT`. I made an attempt to migrate them to `MapReduceIT` since they were just calling the `runTest` method in that IT. But I couldn't get the SSL to work with MapReduceIT and wasn't sure if it was worth the effort.

ctubbsii commented:
I merged main into this to resolve conflicts, and am marking it ready for review. I'm not sure what else needs to be done for this, but that can be ironed out in the review.

ctubbsii commented:
Full ITs passed.
